botcore/utils/__init__.py

```diff
@@ -1,14 +1,32 @@
 """Useful utilities and tools for Discord bot development."""
 
-from botcore.utils import (caching, channel, extensions, logging, members, monkey_patches, regex, scheduling)
+from botcore.utils import (_monkey_patches, caching, channel, extensions, logging, members, regex, scheduling)
```

Comment:
Are the parenthesis necessary?

Reply:
https://github.com/python-discord/bot-core/pull/35/commits/ca9fbeb01fe39867071f6bbd4281411fe28fa060